packages/perspective/plan/view-action-plan.ts

```diff
@@ -250,6 +250,13 @@ export const viewActionPlan =
           },
         });
 
+      case 'actionDutchAuctionSchedule':
+      case 'actionDutchAuctionEnd':
+      case 'actionDutchAuctionWithdraw':
+        return new ActionView({
+          actionView: actionPlan.action,
+        });
```

Comment:
This will need to be modified soon when the `*View` protos for these action types land in core.